src/Repository/CommitteeFeedItemRepository.php

```diff
@@ -82,4 +83,16 @@ private function createCommitteeTimelineQueryBuilder(string $committeeUuid): Que
 
         return $qb;
     }
+
+    public function removeEventItems(BaseEvent $event)
+    {
+        $qb = $this->createQueryBuilder('i');
+        $qb->delete()
```

Comment:
do you really need `expr` here ?

```php
->where('i.itemType = :type')
->andWhere('i.event = :event')
```

isn't enough ?

Reply:
Soz, it's an old habit ;)